identity-account/Cargo.toml

```diff
@@ -27,6 +27,7 @@ zeroize = { version = "1.4", optional = true }
 
 [dev-dependencies]
 futures = { version = "0.3" }
+once_cell = { version = "1.10", default-features = false, features = ["std"] }
```

Comment:
Thanks for the catch. It is used when adding the stronghold test from the PR description in `identity_account::tests::util`, but while that is not actually in the code base this can be removed.